runtime/near-vm-runner/src/wasmer2_runner.rs

```diff
@@ -233,7 +233,7 @@ impl Wasmer2Config {
 //  major version << 6
 //  minor version
 const WASMER2_CONFIG: Wasmer2Config = Wasmer2Config {
-    seed: (1 << 10) | (9 << 6) | 0,
```

Comment:
(also I don’t really get why wasmer2 also needs a bump, if this is only about near-vm)